README.MD

```diff
@@ -13,9 +13,14 @@ This is the Go version of the [onboarding exercise](https://github.com/wexinc/ps
 - [Viper](https://github.com/spf13/viper)
 - [Logrus](https://github.com/Sirupsen/logrus)
 
+## Technical Proposal
+-[Swagger](https://docs.google.com/document/d/1K1WaCZhHi35BbvQ7UAGWOlJeGBoIlLVXpW7EQ8ryFdw/edit?usp=share_link)
+
 ## Pre-requisites
-- Docker
-- Go
+- [Go](https://golang.org/doc/install)
+- [Docker](https://docs.docker.com/manuals/)
+- [Docker-compose](https://docs.docker.com/compose/)
+- [Swagger](https://github.com/swaggo/gin-swagger)
 
 ## Getting started
```

Comment:
These instructions do not work for me. When running make I get
```make: swag: No such file or directory                                                                         
make: *** [makefile:13: run-with-docs] Error 127```
Where on your machine is swag installed?

Reply:
Just need to follow the prerequisite and install swag